tools/link.py

```diff
@@ -1391,6 +1397,7 @@ def phase_linker_setup(options, state, newargs):  # noqa: C901, PLR0912, PLR0915
   settings.SUPPORTS_PROMISE_ANY = feature_matrix.caniuse(feature_matrix.Feature.PROMISE_ANY)
   if not settings.BULK_MEMORY:
     settings.BULK_MEMORY = feature_matrix.caniuse(feature_matrix.Feature.BULK_MEMORY)
+  default_setting('WASM_BIGINT', feature_matrix.caniuse(feature_matrix.Feature.JS_BIGINT_INTEGRATION))
```

Comment:
Tthis looks like it would turn it on even if `WASM=0` disabled it above (since it won't be in user_settings).

Reply:
the `feature_matrix.disable_feature` line above takes care of that. Basically because we have both a setting and the feature matrix that tracks the features, we have to keep them synchronized.